frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/detailOverview.jspf

```diff
@@ -122,6 +122,15 @@
 										</div>
 									</div>
 								</div>
+								<div class="btn-group" role="group" id="clearingRequest">
+									<button id="createClearingRequestBtn" type="button" class="btn btn-secondary <core_rt:if test="${not empty project.clearingRequestId}">d-none</core_rt:if>">
+										<liferay-ui:message key="create.clearing.request" />
+									</button>
+									<button id="viewClearingRequestBtn" type="button" class="btn btn-secondary <core_rt:if test="${empty project.clearingRequestId}">d-none</core_rt:if>"
+										data-clearing-request-id="${project.clearingRequestId}">
+										<liferay-ui:message key="view.clearing.request" />
+									</button>
```

Comment:
Instead of using jstl:if to hide a button. It can be used to add appropriate button . It will prevent presence of unused button hidded in UI

Reply:
This is needed, because once clearing request is created, the `Create Clearing Request` button should be hidden and `View Clearing Request` button should be displayed.
I tried making it work using single button by changing the button `id` and content `html` after CR creation, But it isn't working as expected.